### DISCUSSION TITLE
Where and how does Astroid Cache store files?

### DISCUSSION THREAD

**Discussion by srcarney** [templaza/astroid-framework#903] [General]
Generally I use JCH Optimize Pro to manage the Cache on all of my websites. I turn OFF the Return to Cache System Cache setting from Joomla's Global Configuration-System, and I also turn off the Joomla  System Page Cache plugin. Then JCH cache handles everything. 

With the Astroid Framework, I see that the Astroid Clear Cache button reports "Joomla Cache Cleared" and "Astroid Cache Cleared". I am wondering if the Astroid Cache is stored somewhere separate from the Joomla cache. The reason I am asking is that I generally turn off all caching except for the JCH Optimize Pro Cache. But, I want to make sure that if I turn off the Astroid Clear Cache module, the JCH Optimize Pro cache clearing will also clear the Astroid files. Does anybody here know about this?

**Comment by sonvnn** [comment]
@srcarney Astroid system automatically compiles scss files and generates css. All data will be stored at `ROOT/media/templates/site/TEMPLATENAME/css/compiled-xxxxx.css` Astroid Clear Cache will delete these files to force the system to recompile SCSS again. It does not affect JCH operation. So you do not need to worry about it.

**Reply by srcarney** [reply]
Thanks Sonny for the reply. I had been wondering if I could just have one button to clear the cache. But, I need  to keep the button for removing the Astroid cache for the times when I have been making changes to the template. The cache that JCH will clear will be the content cache.
